vaadin-combo-box-flow-parent/vaadin-combo-box-flow-integration-tests/src/test/java/com/vaadin/flow/component/combobox/test/FilteringIT.java

```diff
@@ -165,6 +169,61 @@ public void configureEmptyFilterToReturnNoItems_useCaseWorks() {
         assertItemsNotLoaded();
     }
 
+    @Test
+    public void unknownItemCountLazyLoadingFiltering_applyFilter_allPagesContainFilteredItems() {
+        clickButton("switch-to-unknown-item-count");
+        verifyFilteredItems(comboBoxWithFilteredItems);
+    }
+
+    @Test
+    public void inMemoryItemsFiltering_applyFilter_allPagesContainFilteredItems() {
+        clickButton("switch-to-in-memory-items");
```

Comment:
Use the package-private constant proposed to be defined in the `FilteringPage.java`.

Reply:
done